debian/control

```diff
@@ -2,7 +2,7 @@ Source: ola
 Priority: optional
 Maintainer: Wouter Verhelst <wouter@debian.org>
 Uploaders: RenZO <renzo@imaginux.com>
-Build-Depends: debhelper (>= 13), autotools-dev, dh-autoreconf, dh-python, bash-completion, libcppunit-dev, bison, flex, pkg-config, uuid-dev, python3, python3-protobuf, libprotobuf-dev, protobuf-compiler, libprotoc-dev, libusb-1.0-0-dev, libftdi1-dev, liblo-dev, libmicrohttpd-dev, libncurses5-dev, libavahi-client-dev, python3-numpy
+Build-Depends: debhelper-compat (= 12), autotools-dev, dh-autoreconf, dh-python, bash-completion, libcppunit-dev, bison, flex, pkg-config, uuid-dev, python3, python3-protobuf, libprotobuf-dev, protobuf-compiler, libprotoc-dev, libusb-1.0-0-dev, libftdi1-dev, liblo-dev, libmicrohttpd-dev, libncurses5-dev, libavahi-client-dev, python3-numpy
```

Comment:
According to [debhelper-compat-upgrade-checklist(7)](https://manpages.debian.org/bullseye-backports/debhelper/debhelper-compat-upgrade-checklist.7.en.html), compat level 13 is currently the recommended mode. We effectively already required debhelper 13 before, so that isn't a regression; I would therefore recommend making the dependency be `debhelper-compat (=13)`. If the concern is that it won't be installable on buster (where the available version is limited to 12), there *is* a debhelper 13 in buster-backports.

Reply:
Oh, wow. I completely forgot about that.

The thing is that in compat level 13, we need to account for every file. If `make install` creates a file but it's not installed in any package and also not explicitly listed as something we don't install, you get an error. Between it being short to release time last time around, and having had to remove the rdm tester which produced a lot of those errors, I decided it wasn't the time for that.

When Debian 12 comes out, I'll be able to re-enable the rdm tester, so that might be a good point to try that again. I'll see if I can do a pull request at that point.

Reply:
> Oh, wow. I completely forgot about that.

Heh, don't worry I really dug through _all_ of the commits on Salsa!

> When Debian 12 comes out, I'll be able to re-enable the rdm tester, so that might be a good point to try that again.

I fixed building the RDM tester in #1812 and it works with Python 3 now. Between that and removing debconf/moving to Systemd I think those should be the last bits removed in the Debian downstream from here.

> I'll see if I can do a pull request at that point.

Amazing!

Reply:
> > Oh, wow. I completely forgot about that.
> 
> Heh, don't worry I really dug through _all_ of the commits on Salsa!

Apparently :-D

> > When Debian 12 comes out, I'll be able to re-enable the rdm tester, so that might be a good point to try that again.
> 
> I fixed building the RDM tester in #1812 and it works with Python 3 now. Between that and removing debconf/moving to Systemd I think those should be the last bits removed in the Debian downstream from here.

I know. The problem though is that I couldn't introduce a new package into Debian anymore at the time, because the freeze had started and Debian's policies don't allow for that anymore then. Once the release has happened, this will not be a problem and it's just a matter of adding the right configuration then.